packages/flutter/lib/src/widgets/page_view.dart

```diff
@@ -581,7 +581,7 @@ class PageView extends StatefulWidget {
   /// child that could possibly be displayed in the page view, instead of just
   /// those children that are actually visible.
   ///
-  /// If the children [List] is going to be mutated,
+  /// If the [children] is going to be mutated,
```

Comment:
Sorry to keep nitpicking here, but this is gramatically dubious now and a little ambiguous.

Perhaps it would be clear to just say:

> Like other widgets in the framework, this widget expects that the [children] list will not be mutated after it has been passed in here. See the documentation at [SliverChildListDelegate.children] for more details.

WDYT?

Reply:
This looks like more professional, 👍